launcher/spec/launch_policy.go

```diff
@@ -35,7 +58,15 @@ func GetLaunchPolicy(imageLabels map[string]string) (LaunchPolicy, error) {
 
 	if v, ok := imageLabels[cmdOverride]; ok {
 		if launchPolicy.AllowedCmdOverride, err = strconv.ParseBool(v); err != nil {
-			return LaunchPolicy{}, fmt.Errorf("value of LABEL %s of the image is not a boolean %s", cmdOverride, v)
+			return LaunchPolicy{}, fmt.Errorf("value of LABEL %s of the image is not a boolean: %s, contact the image author", cmdOverride, v)
```

Comment:
"invalid image LABEL '%s' (not a boolean); contact the image author"

Reply:
done